cmd/pd-server/main.go

```diff
@@ -123,21 +111,25 @@ func createServerWrapper(args []string) (context.Context, context.CancelFunc, ba
 
 	metricutil.Push(&cfg.Metric)
 
-	err = join.PrepareJoinCluster(cfg)
-	if err != nil {
-		log.Fatal("join meet error", errs.ZapError(err))
-	}
-
-	// Creates server.
+	// todo: add more service mode and multi service mode
 	ctx, cancel := context.WithCancel(context.Background())
-	serviceBuilders := []server.HandlerBuilder{api.NewHandler, apiv2.NewV2Handler, swaggerserver.NewHandler, autoscaling.NewHandler}
-	serviceBuilders = append(serviceBuilders, dashboard.GetServiceBuilders()...)
-	svr, err := server.CreateServer(ctx, cfg, serviceBuilders...)
+	m, err := config.ParseServiceMode(cfg.ServiceModes)
```

Comment:
In my understanding, based on our last discussion, config.ParseServiceMode will be switched to parse subcommand at the beginning of main(), correct?

Reply:
It will be did in another pr. This commit only sync with master

Reply:
Thanks for the confirmation